src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -98,18 +100,19 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
   const resetOrder = async () => {
     const unsetFulfillmentOptions =
       await unsetFulfillmentOptionMutation.submitMutation({
-        variables: {
-          input: {
-            id: orderData.internalID,
-          },
-        },
+        variables: { input: { id: orderData.internalID } },
       })
 
-    const validatedResult = validateAndExtractOrderResponse(
+    const unsetPaymentMethod = await unsetPaymentMethodMutation.submitMutation({
+      variables: { input: { id: orderData.internalID } },
+    })
+
+    validateAndExtractOrderResponse(
       unsetFulfillmentOptions.unsetOrderFulfillmentOption?.orderOrError,
     )
-
-    return validatedResult
```

Comment:
I removed this since I didn't see that it was used in the code

Reply:
Ah yeah, i think early on we were using it when opening express checkout so we wanted the updated order totals/shipping rates at the end